packages/mask/background/tasks/NotificationsToMobile.ts

```diff
@@ -0,0 +1,10 @@
+import { notify } from 'async-call-rpc/full'
```

Comment:
This file already appears in `packages/mask/background/tasks/NotificationsToMobile.ts`. Please update with develop branch.